.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+.idea
```

Comment:
We will keep this for now. But a practice is to have in .gitignore only files that a generated by the project.
All files generated by different IDEs must be ignored globally

https://github.com/github/gitignore/tree/master/Global
https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer
https://augustl.com/blog/2009/global_gitignores/